u8timeseries/tests/test_autoregression_models.py

```diff
@@ -3,31 +3,67 @@
 import numpy as np
 
 from ..timeseries import TimeSeries
+from ..utils import timeseries_generation as tg
+from ..metrics import mape, overall_percentage_error, mase
 from u8timeseries import Prophet, KthValueAgoBaseline, ExponentialSmoothing, TimeSeries, Arima, AutoArima
 from u8timeseries.models.theta import Theta
 
-class ModelsTestCase(unittest.TestCase):
+class AutoregressionModelsTestCase(unittest.TestCase):
 
-    # forecasting horizon used in predictions
+    # forecasting horizon used in runnability tests
     forecasting_horizon = 5
 
-    # dummy timeseries for autoregression forecasting
-    times = pd.date_range('20000101', '20000130')
-    values = np.sin(range(len(times))) + np.array([2.0] * len(times))
-    ts: TimeSeries = TimeSeries.from_times_and_values(times, values)
+    # dummy timeseries for runnability tests
+    np.random.seed(0)
```

Comment:
`0` stands for random seed right? I think you can use another value, for reproducibility and making the tests deterministic.

Reply:
done